test/store_unittest.cc

```diff
@@ -90,123 +90,141 @@ TEST_F(MetadataStoreTest, UpdateResourceDoesNotUpdateMetadata) {
 }
 
 TEST_F(MetadataStoreTest, UpdateMetadataChangesMetadataMap) {
-  MonitoredResource resource("type", {});
+  const std::string frn = "/type";
```

Comment:
`frn` is an abbreviation, which is discouraged by the [style guide](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules). Frankly, I would just keep the `resource` variable name. Also everywhere below.

Reply:
Done

Reply:
Done